Index: src/twitux-app.c
===================================================================
--- src/twitux-app.c	(revision 369)
+++ src/twitux-app.c	(working copy)
@@ -1467,7 +1467,10 @@
                       GtkTreeIter  iter)
 {
 	GtkListStore *store;
-	GdkPixbuf	 *pixbuf;
+	/* By Kaity G. B. <uberChick-at-uberChicGeekChick.Com>		*
+	 *	2009-01-30:Begin fixed sized avatar patch; part 2.1.	*/
+	GdkPixbuf	 *pixbuf, *resized;
+	/* End fixed sized avatar patch; part 2.1.			 */
 	GError		 *error = NULL;
 
 	pixbuf = gdk_pixbuf_new_from_file (file, &error);
@@ -1478,11 +1481,15 @@
 		g_error_free (error);
 		return;
 	}
-	
+	/* By Kaity G. B. <uberChick-at-uberChicGeekChick.Com>		*
+	 *	2009-01-30:Begin fixed sized avatar patch; part 2.2.	*/
+	resized = gdk_pixbuf_scale_simple( pixbuf, 48, 48, GDK_INTERP_BILINEAR );
+	/* End fixed sized avatar patch; part 2.2.			 */
+
 	store = twitux_tweet_list_get_store ();
 
-	gtk_list_store_set (store, &iter,
-						PIXBUF_AVATAR, pixbuf, -1);
+	gtk_list_store_set (store, &iter, PIXBUF_AVATAR, resized, -1);
+	g_object_unref(pixbuf);//memory clean-up: Kaity G. B.
 }
 
 void
@@ -1493,6 +1500,7 @@
 {
 	TwituxAppPriv *priv;
 	gchar 		  *title_text;
+	GdkPixbuf	*resized;
 	
 	priv = GET_PRIV (app);
 
@@ -1503,7 +1511,12 @@
 	g_free (title_text);
 	
 	if (pixbuf) {
-		gtk_image_set_from_pixbuf (GTK_IMAGE (priv->expand_image), pixbuf);
+		/* By Kaity G. B. <uberChick-at-uberChicGeekChick.Com>		*
+		 *	2009-01-30:Begin fixed sized avatar patch; part 3.	*/
+		resized = gdk_pixbuf_scale_simple( pixbuf, 73, 73, GDK_INTERP_BILINEAR );
+		/* End fixed sized avatar patch; part 3.			 */
+		gtk_image_set_from_pixbuf (GTK_IMAGE (priv->expand_image), resized);
+		g_object_unref(resized);//memory clean-up: Kaity G. B.
 	}
 	
 	gtk_widget_show (priv->expand_box);
Index: src/twitux-tweet-list.c
===================================================================
--- src/twitux-tweet-list.c	(revision 369)
+++ src/twitux-tweet-list.c	(working copy)
@@ -147,36 +147,31 @@
 
 	priv = GET_PRIV (list);
 	
-	g_object_set (list,
-				  "rules-hint", TRUE,
-				  "reorderable", FALSE,
-				  "headers-visible", FALSE,
-				  NULL);
+	g_object_set( list,
+			"rules-hint", TRUE,
+			"reorderable", FALSE,
+			"headers-visible", FALSE,
+			NULL
+	);
 
-	renderer = gtk_cell_renderer_pixbuf_new ();
-	avatar_column =
-		gtk_tree_view_column_new_with_attributes (NULL,
-												  renderer,
-												  "pixbuf", PIXBUF_AVATAR,
-												  NULL);
-
-	gtk_tree_view_append_column (GTK_TREE_VIEW (list), avatar_column);
-
-	renderer = gtk_cell_renderer_text_new ();
-	tweet_column =
-		gtk_tree_view_column_new_with_attributes (NULL,
-												  renderer,
-												  "markup", STRING_TEXT,
-												  NULL);
-	gtk_tree_view_column_set_sizing (tweet_column,
-									 GTK_TREE_VIEW_COLUMN_FIXED);
-	g_object_set (renderer,
-				  "ypad", 0,
-				  "xpad", 5,
-				  "yalign", 0.0,
-				  "wrap-mode", PANGO_WRAP_WORD_CHAR,
-				  NULL);
-
+	/* By Kaity G. B. <uberChick-at-uberChicGeekChick.Com>		*
+	 *	2009-01-30:Begin fixed sized avatar patch; part 1.	*/
+	renderer = gtk_cell_renderer_pixbuf_new();
+	gtk_cell_renderer_set_fixed_size(renderer, 48, 48);
+	avatar_column=gtk_tree_view_column_new_with_attributes(
+				NULL, renderer,"pixbuf", PIXBUF_AVATAR, NULL
+			);
+	gtk_tree_view_column_set_sizing(avatar_column, GTK_TREE_VIEW_COLUMN_FIXED);
+	gtk_tree_view_column_set_min_width(avatar_column, 48);
+	gtk_tree_view_column_set_fixed_width(avatar_column, 48);
+	gtk_tree_view_append_column(GTK_TREE_VIEW (list), avatar_column);
+	/* End fixed sized avatar patch; part 1.			 */
+	
+	
+	renderer = gtk_cell_renderer_text_new();
+	tweet_column = gtk_tree_view_column_new_with_attributes (NULL, renderer, "markup", STRING_TEXT, NULL);
+	gtk_tree_view_column_set_sizing (tweet_column, GTK_TREE_VIEW_COLUMN_FIXED);
+	g_object_set (renderer, "ypad", 0,"xpad", 50, "yalign", 0.0,"wrap-mode", PANGO_WRAP_WORD_CHAR, NULL);
 	gtk_tree_view_append_column (GTK_TREE_VIEW (list), tweet_column);
 	
 	priv->text_column = tweet_column;
@@ -220,10 +215,11 @@
 						-1);
 
 	twitux_app_expand_message (name, date, tweet, pixbuf);
-
+	
 	g_free (name);
 	g_free (tweet);
 	g_free (date);
+	g_object_unref(pixbuf);//memory clean-up: Kaity G. B.
 }
 
 static void
