Index: AUTHORS
===================================================================
--- AUTHORS	(revision 372)
+++ AUTHORS	(working copy)
@@ -7,7 +7,10 @@
 
  * Brian Pepple  <bpepple@fedoraproject.org>
 
+--- Direct Messaging, Avatar size/message scrolling, memory leaks patches/fixes.
 
+ * Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
 --- Translators:
 
  * Swedish - Daniel Nylander <po@danielnylander.se>
Index: src/twitux-app.c
===================================================================
--- src/twitux-app.c	(revision 372)
+++ src/twitux-app.c	(working copy)
@@ -20,6 +20,7 @@
  *
  * Authors: Brian Pepple <bpepple@fedoraproject.org>
  *			Daniel Morales <daniminas@gmail.com>
+ *			Kaity G. B. <KaityGB@uberChicGeekChick.Com>
  */
 
 #include <config.h>
@@ -1466,7 +1467,6 @@
 twitux_app_set_image (const gchar *file,
                       GtkTreeIter  iter)
 {
-	GtkListStore *store;
 	GdkPixbuf	 *pixbuf, *resized;
 	GError		 *error = NULL;
 
@@ -1479,10 +1479,9 @@
 		return;
 	}
 
-	resized = gdk_pixbuf_scale_simple(pixbuf, 48, 48, GDK_INTERP_BILINEAR);
+	resized = gdk_pixbuf_scale_simple( pixbuf, 48, 48, GDK_INTERP_BILINEAR );
 
-	store = twitux_tweet_list_get_store ();
-	gtk_list_store_set (store, &iter, PIXBUF_AVATAR, resized, -1);
+	gtk_list_store_set ( (twitux_tweet_list_get_store()) , &iter, PIXBUF_AVATAR, resized, -1);
 	g_object_unref(pixbuf);
 }
 
@@ -1505,8 +1504,8 @@
 	g_free (title_text);
 	
 	if (pixbuf) {
-		resized = gdk_pixbuf_scale_simple(pixbuf, 73, 73, GDK_INTERP_BILINEAR);
-		gtk_image_set_from_pixbuf (GTK_IMAGE (priv->expand_image), resized);
+		resized = gdk_pixbuf_scale_simple( pixbuf, 73, 73, GDK_INTERP_BILINEAR );
+		gtk_image_set_from_pixbuf(GTK_IMAGE (priv->expand_image), resized);
 		g_object_unref(resized);
 	}
 	
Index: src/twitux-tweet-list.c
===================================================================
--- src/twitux-tweet-list.c	(revision 372)
+++ src/twitux-tweet-list.c	(working copy)
@@ -19,6 +19,7 @@
  *
  * Authors: Brian Pepple <bpepple@fedoraproject.org>
  *			Daniel Morales <daniminas@gmail.com>
+ *			Kaity G. B. <KaityGB@uberChicGeekChick.Com>
  */
 
 #include "config.h"
@@ -152,33 +153,21 @@
 				 "reorderable", FALSE,
 				 "headers-visible", FALSE,
 				 NULL);
-
+	
 	renderer = gtk_cell_renderer_pixbuf_new();
-	gtk_cell_renderer_set_fixed_size (renderer, 48, 48);
-	avatar_column =
-		gtk_tree_view_column_new_with_attributes (NULL,
-												  renderer,
-												  "pixbuf", PIXBUF_AVATAR,
-												  NULL);
-	gtk_tree_view_column_set_sizing (avatar_column, GTK_TREE_VIEW_COLUMN_FIXED);
-	gtk_tree_view_column_set_min_width (avatar_column, 48);
-	gtk_tree_view_column_set_fixed_width (avatar_column, 48);
-	gtk_tree_view_append_column (GTK_TREE_VIEW (list),
-								 avatar_column);
+	gtk_cell_renderer_set_fixed_size(renderer, 48, 48);
+	avatar_column=gtk_tree_view_column_new_with_attributes(
+				NULL, renderer,"pixbuf", PIXBUF_AVATAR, NULL
+			);
+	gtk_tree_view_column_set_sizing(avatar_column, GTK_TREE_VIEW_COLUMN_FIXED);
+	gtk_tree_view_column_set_min_width(avatar_column, 48);
+	gtk_tree_view_column_set_fixed_width(avatar_column, 48);
+	gtk_tree_view_append_column(GTK_TREE_VIEW (list), avatar_column);
 	
 	renderer = gtk_cell_renderer_text_new();
-	tweet_column =
-		gtk_tree_view_column_new_with_attributes (NULL,
-												  renderer,
-												  "markup", STRING_TEXT,
-												  NULL);
+	tweet_column = gtk_tree_view_column_new_with_attributes (NULL, renderer, "markup", STRING_TEXT, NULL);
 	gtk_tree_view_column_set_sizing (tweet_column, GTK_TREE_VIEW_COLUMN_FIXED);
-	g_object_set (renderer,
-				  "ypad", 0,
-				  "xpad", 5,
-				  "yalign", 0.0,
-				  "wrap-mode", PANGO_WRAP_WORD_CHAR,
-				  NULL);
+	g_object_set (renderer, "ypad", 0,"xpad", 50, "yalign", 0.0,"wrap-mode", PANGO_WRAP_WORD_CHAR, NULL);
 	gtk_tree_view_append_column (GTK_TREE_VIEW (list), tweet_column);
 	
 	priv->text_column = tweet_column;
Index: ChangeLog
===================================================================
--- ChangeLog	(revision 372)
+++ ChangeLog	(working copy)
@@ -1,10 +1,58 @@
-2009-02-11  Brian Pepple  <bpepple@fedoraproject.org>
+2009-02-11  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
 
-	* src/twitux-app.c:
-	* src/twitux-tweet-list.c:
-	Plug a leak with some pixbufs, and set the size of
-	avatar pixbuf.  (Patch provided by Kaity G. B.)
+	* po/Makefile.in: PO_LINGUAS & USER_LINGUAS 
+		I added export GREP_OPTIONS=""; to both shell execs so it
+		actually finds files in the format it expects.
+	
+	* data/Makefile.in: twitux.desktop
+		changed:
+		twitux.desktop.in: $(top_builddir)/config.status $(srcdir)/twitux.desktop.in.in
+		to
+		twitux.desktop: $(top_builddir)/config.status $(srcdir)/twitux.desktop.in
 
+
+2009-02-11  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
+	* configure: ac_config_files
+		I modified them so they look for 'po/Makefile' and
+		'data/twitux.desktop' correctly an will fall back to the
+		correct '.in' when needed.
+
+	* po/Makefile.in: clean & dist distdirs
+		I corrected the sh commands that find gettext .po, .gmo, & etc
+		files.  Now po/Makefile's clean & beyond will now work.  Most
+		importantly make clean will now work.
+	
+	* po/POFILES: twitux.desktop.in.in
+	* po/POFILES.in: twitux.desktop.in.in
+		These were both referencing twitux.desktop.in.in which was
+		only a duplicate of twitux.desktop.in.  They now correctly
+		look for twitux.desktop.in
+
+	* data/Makefile.am: .PRECIOUS: Makefile and multiple varibles
+	* data/Makefile.in: .PRECIOUS: Makefile and multiple varibles
+		I changed references to Makefile.in.in &
+		twitux-desktop.in.in to point to Makefile.in &
+		twitux.desktop.in respectfully.  I also removed the unnead
+		'.in.in' files.
+
+2009-02-09  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
+	* src/twitux-app.c: twitux_app_set_image & twitux_app_expand_message
+	* src/twitux-tweet-list.c: tweet_list_changed_cb
+		I plugged some large memory leaks by unreferencing
+		GdkPixbuf(s) & various other gobjects used my Twitux for
+		Twitter's avatars.
+
+2009-02-06  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
+	* src/twitux-app.c: twitux_app_set_image & twitux_app_expand_message
+	* src/twitux-tweet-list.c: tweet_list_setup_view
+		I wrote resizing code to stop large images from bloating
+		Twitux's tweet list.  I have also added code to keep the
+		tweet list avatar column at a fixed width.
+
+
 2009-01-31  Daniel Morales  <daniminas@gmail.com>
 
 	* src/twitux-network.c: Bugfix to Add/Remove friends methods,
Index: data/Makefile.am
===================================================================
--- data/Makefile.am	(revision 372)
+++ data/Makefile.am	(working copy)
@@ -1,6 +1,6 @@
 desktopdir 	 = $(datadir)/applications
-desktop_in_files = twitux.desktop.in.in
-desktop_DATA 	 = $(desktop_in_files:.desktop.in.in=.desktop)
+desktop_in_files = twitux.desktop.in
+desktop_DATA 	 = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
 
 schemasdir	 = $(GCONF_SCHEMA_FILE_DIR)
Index: po/pt.po
===================================================================
--- po/pt.po	(revision 372)
+++ po/pt.po	(working copy)
@@ -138,7 +138,7 @@
 msgid "Twitter Account"
 msgstr "Conta Twitter"
 
-#: ../data/main.glade.h:30 ../data/twitux.desktop.in.in.h:2
+#: ../data/main.glade.h:30 ../data/twitux.desktop.in.h:2
 #: ../src/twitux-main.c:46 ../src/twitux-preferences.c:416
 msgid "Twitux"
 msgstr "Twitux"
@@ -227,7 +227,7 @@
 msgid "gtk-quit"
 msgstr "gtk-quit"
 
-#: ../data/twitux.desktop.in.in.h:1
+#: ../data/twitux.desktop.in.h:1
 msgid "A Twitter client"
 msgstr "Um cliente Twitter"
 
Index: po/es.po
===================================================================
--- po/es.po	(revision 372)
+++ po/es.po	(working copy)
@@ -101,7 +101,7 @@
 msgid "Send Direct Message"
 msgstr "Enviar Mensajes Directos"
 
-#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.in.h:2
+#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.h:2
 #: ../src/twitux-main.c:46 ../src/twitux-preferences.c:430
 msgid "Twitux"
 msgstr "Twitux"
@@ -243,7 +243,7 @@
 msgid "_Show Twitux"
 msgstr "_Mostrar Twitux"
 
-#: ../data/twitux.desktop.in.in.h:1
+#: ../data/twitux.desktop.in.h:1
 msgid "A Twitter client"
 msgstr "Un cliente Twitter"
 
Index: po/ja.po
===================================================================
--- po/ja.po	(revision 372)
+++ po/ja.po	(working copy)
@@ -99,7 +99,7 @@
 msgid "Send Direct Message"
 msgstr "ダイレクトメッセージ送信"
 
-#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.in.h:2
+#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.h:2
 #: ../src/twitux-main.c:46 ../src/twitux-preferences.c:430
 msgid "Twitux"
 msgstr ""
@@ -241,7 +241,7 @@
 msgid "_Show Twitux"
 msgstr "Twituxを表示(_S)"
 
-#: ../data/twitux.desktop.in.in.h:1
+#: ../data/twitux.desktop.in.h:1
 msgid "A Twitter client"
 msgstr "Twitterクライアント"
 
Index: po/POTFILES.in
===================================================================
--- po/POTFILES.in	(revision 372)
+++ po/POTFILES.in	(working copy)
@@ -8,7 +8,7 @@
 [type: gettext/glade]data/prefs_dlg.xml
 [type: gettext/glade]data/send_message_dlg.xml
 [type: gettext/glade]data/spell_dlg.xml
-data/twitux.desktop.in.in
+data/twitux.desktop.in
 data/twitux.schemas.in
 
 src/twitux-about.c
Index: AUTHORS
===================================================================
--- AUTHORS	(revision 372)
+++ AUTHORS	(working copy)
@@ -7,7 +7,10 @@
 
  * Brian Pepple  <bpepple@fedoraproject.org>
 
+--- Direct Messaging, Avatar size/message scrolling, memory leaks patches/fixes.
 
+ * Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
 --- Translators:
 
  * Swedish - Daniel Nylander <po@danielnylander.se>
Index: ChangeLog
===================================================================
--- ChangeLog	(revision 372)
+++ ChangeLog	(working copy)
@@ -1,10 +1,59 @@
-2009-02-11  Brian Pepple  <bpepple@fedoraproject.org>
+2009-02-11  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
 
-	* src/twitux-app.c:
-	* src/twitux-tweet-list.c:
-	Plug a leak with some pixbufs, and set the size of
-	avatar pixbuf.  (Patch provided by Kaity G. B.)
+	* po/Makefile.in: PO_LINGUAS & USER_LINGUAS 
+		I added export GREP_OPTIONS=""; to both shell execs so it
+		actually finds files in the format it expects.
+	
+	* data/Makefile.in: twitux.desktop
+		changed:
+		twitux.desktop.in: $(top_builddir)/config.status $(srcdir)/twitux.desktop.in.in
+		to
+		twitux.desktop: $(top_builddir)/config.status $(srcdir)/twitux.desktop.in
+	
+	* help/twitux-omf.in: depricated.
 
+2009-02-11  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
+	* configure: ac_config_files
+		I modified them so they look for 'po/Makefile' and
+		'data/twitux.desktop' correctly an will fall back to the
+		correct '.in' when needed.
+
+	* po/Makefile.in: clean & dist distdirs
+		I corrected the sh commands that find gettext .po, .gmo, & etc
+		files.  Now po/Makefile's clean & beyond will now work.  Most
+		importantly make clean will now work.
+	
+	* po/POFILES: twitux.desktop.in.in
+	* po/POFILES.in: twitux.desktop.in.in
+		These were both referencing twitux.desktop.in.in which was
+		only a duplicate of twitux.desktop.in.  They now correctly
+		look for twitux.desktop.in
+
+	* data/Makefile.am: .PRECIOUS: Makefile and multiple varibles
+	* data/Makefile.in: .PRECIOUS: Makefile and multiple varibles
+		I changed references to Makefile.in.in &
+		twitux-desktop.in.in to point to Makefile.in &
+		twitux.desktop.in respectfully.  I also removed the unnead
+		'.in.in' files.
+
+2009-02-09  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
+	* src/twitux-app.c: twitux_app_set_image & twitux_app_expand_message
+	* src/twitux-tweet-list.c: tweet_list_changed_cb
+		I plugged some large memory leaks by unreferencing
+		GdkPixbuf(s) & various other gobjects used my Twitux for
+		Twitter's avatars.
+
+2009-02-06  Kaity G. B. <KaityGB-at-uberChicGeekChick.Com>
+
+	* src/twitux-app.c: twitux_app_set_image & twitux_app_expand_message
+	* src/twitux-tweet-list.c: tweet_list_setup_view
+		I wrote resizing code to stop large images from bloating
+		Twitux's tweet list.  I have also added code to keep the
+		tweet list avatar column at a fixed width.
+
+
 2009-01-31  Daniel Morales  <daniminas@gmail.com>
 
 	* src/twitux-network.c: Bugfix to Add/Remove friends methods,
Index: src/twitux-app.c
===================================================================
--- src/twitux-app.c	(revision 372)
+++ src/twitux-app.c	(working copy)
@@ -20,6 +20,7 @@
  *
  * Authors: Brian Pepple <bpepple@fedoraproject.org>
  *			Daniel Morales <daniminas@gmail.com>
+ *			Kaity G. B. <KaityGB@uberChicGeekChick.Com>
  */
 
 #include <config.h>
@@ -1466,7 +1467,6 @@
 twitux_app_set_image (const gchar *file,
                       GtkTreeIter  iter)
 {
-	GtkListStore *store;
 	GdkPixbuf	 *pixbuf, *resized;
 	GError		 *error = NULL;
 
@@ -1479,10 +1479,9 @@
 		return;
 	}
 
-	resized = gdk_pixbuf_scale_simple(pixbuf, 48, 48, GDK_INTERP_BILINEAR);
+	resized = gdk_pixbuf_scale_simple( pixbuf, 48, 48, GDK_INTERP_BILINEAR );
 
-	store = twitux_tweet_list_get_store ();
-	gtk_list_store_set (store, &iter, PIXBUF_AVATAR, resized, -1);
+	gtk_list_store_set ( (twitux_tweet_list_get_store()) , &iter, PIXBUF_AVATAR, resized, -1);
 	g_object_unref(pixbuf);
 }
 
@@ -1505,8 +1504,8 @@
 	g_free (title_text);
 	
 	if (pixbuf) {
-		resized = gdk_pixbuf_scale_simple(pixbuf, 73, 73, GDK_INTERP_BILINEAR);
-		gtk_image_set_from_pixbuf (GTK_IMAGE (priv->expand_image), resized);
+		resized = gdk_pixbuf_scale_simple( pixbuf, 73, 73, GDK_INTERP_BILINEAR );
+		gtk_image_set_from_pixbuf(GTK_IMAGE (priv->expand_image), resized);
 		g_object_unref(resized);
 	}
 	
Index: src/twitux-tweet-list.c
===================================================================
--- src/twitux-tweet-list.c	(revision 372)
+++ src/twitux-tweet-list.c	(working copy)
@@ -19,6 +19,7 @@
  *
  * Authors: Brian Pepple <bpepple@fedoraproject.org>
  *			Daniel Morales <daniminas@gmail.com>
+ *			Kaity G. B. <KaityGB@uberChicGeekChick.Com>
  */
 
 #include "config.h"
@@ -152,33 +153,21 @@
 				 "reorderable", FALSE,
 				 "headers-visible", FALSE,
 				 NULL);
-
+	
 	renderer = gtk_cell_renderer_pixbuf_new();
-	gtk_cell_renderer_set_fixed_size (renderer, 48, 48);
-	avatar_column =
-		gtk_tree_view_column_new_with_attributes (NULL,
-												  renderer,
-												  "pixbuf", PIXBUF_AVATAR,
-												  NULL);
-	gtk_tree_view_column_set_sizing (avatar_column, GTK_TREE_VIEW_COLUMN_FIXED);
-	gtk_tree_view_column_set_min_width (avatar_column, 48);
-	gtk_tree_view_column_set_fixed_width (avatar_column, 48);
-	gtk_tree_view_append_column (GTK_TREE_VIEW (list),
-								 avatar_column);
+	gtk_cell_renderer_set_fixed_size(renderer, 48, 48);
+	avatar_column=gtk_tree_view_column_new_with_attributes(
+				NULL, renderer,"pixbuf", PIXBUF_AVATAR, NULL
+			);
+	gtk_tree_view_column_set_sizing(avatar_column, GTK_TREE_VIEW_COLUMN_FIXED);
+	gtk_tree_view_column_set_min_width(avatar_column, 48);
+	gtk_tree_view_column_set_fixed_width(avatar_column, 48);
+	gtk_tree_view_append_column(GTK_TREE_VIEW (list), avatar_column);
 	
 	renderer = gtk_cell_renderer_text_new();
-	tweet_column =
-		gtk_tree_view_column_new_with_attributes (NULL,
-												  renderer,
-												  "markup", STRING_TEXT,
-												  NULL);
+	tweet_column = gtk_tree_view_column_new_with_attributes (NULL, renderer, "markup", STRING_TEXT, NULL);
 	gtk_tree_view_column_set_sizing (tweet_column, GTK_TREE_VIEW_COLUMN_FIXED);
-	g_object_set (renderer,
-				  "ypad", 0,
-				  "xpad", 5,
-				  "yalign", 0.0,
-				  "wrap-mode", PANGO_WRAP_WORD_CHAR,
-				  NULL);
+	g_object_set (renderer, "ypad", 0,"xpad", 50, "yalign", 0.0,"wrap-mode", PANGO_WRAP_WORD_CHAR, NULL);
 	gtk_tree_view_append_column (GTK_TREE_VIEW (list), tweet_column);
 	
 	priv->text_column = tweet_column;
Index: data/Makefile.am
===================================================================
--- data/Makefile.am	(revision 372)
+++ data/Makefile.am	(working copy)
@@ -1,6 +1,6 @@
 desktopdir 	 = $(datadir)/applications
-desktop_in_files = twitux.desktop.in.in
-desktop_DATA 	 = $(desktop_in_files:.desktop.in.in=.desktop)
+desktop_in_files = twitux.desktop.in
+desktop_DATA 	 = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
 
 schemasdir	 = $(GCONF_SCHEMA_FILE_DIR)
Index: po/pt.po
===================================================================
--- po/pt.po	(revision 372)
+++ po/pt.po	(working copy)
@@ -138,7 +138,7 @@
 msgid "Twitter Account"
 msgstr "Conta Twitter"
 
-#: ../data/main.glade.h:30 ../data/twitux.desktop.in.in.h:2
+#: ../data/main.glade.h:30 ../data/twitux.desktop.in.h:2
 #: ../src/twitux-main.c:46 ../src/twitux-preferences.c:416
 msgid "Twitux"
 msgstr "Twitux"
@@ -227,7 +227,7 @@
 msgid "gtk-quit"
 msgstr "gtk-quit"
 
-#: ../data/twitux.desktop.in.in.h:1
+#: ../data/twitux.desktop.in.h:1
 msgid "A Twitter client"
 msgstr "Um cliente Twitter"
 
Index: po/es.po
===================================================================
--- po/es.po	(revision 372)
+++ po/es.po	(working copy)
@@ -101,7 +101,7 @@
 msgid "Send Direct Message"
 msgstr "Enviar Mensajes Directos"
 
-#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.in.h:2
+#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.h:2
 #: ../src/twitux-main.c:46 ../src/twitux-preferences.c:430
 msgid "Twitux"
 msgstr "Twitux"
@@ -243,7 +243,7 @@
 msgid "_Show Twitux"
 msgstr "_Mostrar Twitux"
 
-#: ../data/twitux.desktop.in.in.h:1
+#: ../data/twitux.desktop.in.h:1
 msgid "A Twitter client"
 msgstr "Un cliente Twitter"
 
Index: po/ja.po
===================================================================
--- po/ja.po	(revision 372)
+++ po/ja.po	(working copy)
@@ -99,7 +99,7 @@
 msgid "Send Direct Message"
 msgstr "ダイレクトメッセージ送信"
 
-#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.in.h:2
+#: ../data/main_window.xml.h:10 ../data/twitux.desktop.in.h:2
 #: ../src/twitux-main.c:46 ../src/twitux-preferences.c:430
 msgid "Twitux"
 msgstr ""
@@ -241,7 +241,7 @@
 msgid "_Show Twitux"
 msgstr "Twituxを表示(_S)"
 
-#: ../data/twitux.desktop.in.in.h:1
+#: ../data/twitux.desktop.in.h:1
 msgid "A Twitter client"
 msgstr "Twitterクライアント"
 
Index: po/POTFILES.in
===================================================================
--- po/POTFILES.in	(revision 372)
+++ po/POTFILES.in	(working copy)
@@ -8,7 +8,7 @@
 [type: gettext/glade]data/prefs_dlg.xml
 [type: gettext/glade]data/send_message_dlg.xml
 [type: gettext/glade]data/spell_dlg.xml
-data/twitux.desktop.in.in
+data/twitux.desktop.in
 data/twitux.schemas.in
 
 src/twitux-about.c
